MidasCivil_Adapter/CRUD/Read/Results/MeshResults.cs

```diff
@@ -98,12 +98,60 @@ private IEnumerable<IResult> ExtractMeshForce(List<int> ids, List<string> loadca
 
         private IEnumerable<IResult> ExtractMeshStress(List<int> ids, List<string> loadcaseIds, MeshResultLayer meshResultLayer)
         {
-            return null;
+            /***************************************************/
+            string filePath = directory + "\\Plate Stress(L).xls";
+            string csvPath = ExcelToCsv(filePath);
+            List<string> meshStressText = File.ReadAllLines(csvPath).ToList();
+            List<MeshStress> meshStresses = new List<MeshStress>();
+
+            List<string> meshStress = meshStressText[10].Split(',').ToList();
+            List<string> InitialmeshStress = meshStress;
```

Comment:
This object never gets updated, which is why in the test script you get 60 entries for Node 1, instead of 30 for Plate 1, 30 for Plate 2 and 30 for Plate 3. Every second entry in Plate 2 and Plate 3 becomes Plate 1.